### DISCUSSION TITLE
The content of a finding does not load.

### DISCUSSION THREAD

**Discussion by ghost** [solodit/feedback#30] [Bug Report] [answered]
Hello I was doing some research on the platform and stumble upon this finding that does not load:

https://solodit.xyz/issues/11-solmate-safetransfer-and-safetransferfrom-does-not-check-the-codesize-of-the-token-address-which-may-lead-to-fund-loss-code4rena-astaria-astaria-contest-git

**Comment by wang1252** [comment, marked as answer]
@pacelliv 

The url of the finding is changed to the following.
https://solodit.xyz/issues/m-16-solmate-safetransfer-and-safetransferfrom-does-not-check-the-code-size-of-the-token-address-which-may-lead-to-funding-loss-sherlock-bond-bond-protocol-git

Thank you for your report.
